packages/base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ckb-lumos/base",
-  "version": "0.12.0",
+  "version": "0.12.1",
```

Comment:
For a breaking change, we need to bump the major version.

Reply:
I think it would be ok to bump the major version before next release.

Reply:
Bumped it to v0.13.0-rc1  :)